test/src/operation.spec.ts

```diff
@@ -0,0 +1,192 @@
+import expect from 'expect';
+import {Operation} from 'puppeteer-core/internal/common/Operation.js';
```

Comment:
it should be possible to import from `puppeteer-core` now

Reply:
The `create` method is an internal only method.